src/questions/questions.controller.ts

```diff
@@ -52,7 +108,88 @@ export class QuestionController {
     );
   }
 
-  @Roles(RoleEnum.MANAGER)
+  @Roles(RoleEnum.MANAGER, RoleEnum.ADMIN)
+  @ApiBearerAuth()
+  @UseGuards(JwtAuthGuard, RoleGuard)
+  @Patch('update-question-stack/:stack_id')
+  @ApiBody({ type: UpdateQuestionStackDto })
+  async updateQuestionStack(
+    @Param('stack_id') stack_id: string,
+    @Body() updatedFields: UpdateQuestionStackDto,
+  ) {
+    const [data, err] = await this.questionService.updateQuestionStack(
+      stack_id,
+      updatedFields,
+    );
+
+    if (!data)
+      return new ResponseObject(
+        HttpStatus.BAD_REQUEST,
+        'Update Question Stack Failed!',
+        data,
+        err,
+      );
+    return new ResponseObject(
+      HttpStatus.OK,
+      'Update Question Stack Successful!',
+      data,
+      err,
+    );
+  }
+
+  @Roles(RoleEnum.MANAGER, RoleEnum.ADMIN)
+  @ApiBearerAuth()
+  @UseGuards(JwtAuthGuard, RoleGuard)
+  @Delete('remove-question-stack/:stack_id')
+  async removeQuestionStack(@Param('stack_id') stack_id: string) {
+    const [data, err] = await this.questionService.removeQuestionStackById(
+      stack_id,
+    );
+
+    if (!data)
+      return new ResponseObject(
+        HttpStatus.BAD_REQUEST,
+        'Remove Question Stack Failed!',
+        data,
+        err,
+      );
+    return new ResponseObject(
+      HttpStatus.OK,
+      'Remove Question Stack Successful!',
+      data,
+      err,
+    );
+  }
+
+  //                   //
+  /*-----Questions-----*/
+  //                   //
+
+  @Roles(RoleEnum.MANAGER, RoleEnum.ADMIN)
+  @Get('get-question/:question_id')
+  @ApiBearerAuth()
+  @UseGuards(JwtAuthGuard, RoleGuard)
+  async findOneQuestion(@Param('question_id') question_id: string) {
+    const [data, err] = await this.questionService.findOneQuestionById(
+      question_id,
+    );
+
+    if (!data)
+      return new ResponseObject(
+        HttpStatus.BAD_REQUEST,
+        'Find A Question Failed!',
+        data,
+        err,
+      );
+    return new ResponseObject(
+      HttpStatus.OK,
+      'Find A Question Successful!',
+      data,
+      err,
+    );
+  }
+
+  @Roles(RoleEnum.MANAGER, RoleEnum.ADMIN)
   @Post('create-question')
```

Comment:
no need to add 'create-question' because when FE see questions and with post, they will understand create questions